Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionAutomatedALMDefects.cs

```diff
@@ -107,6 +107,9 @@ public override void Execute(ReportInfo RI)
                             currentALMDefectFieldsValues.Add("Summary", defectSuggestion.Summary);
                             currentALMDefectFieldsValues.Add("description", defectSuggestion.ErrorDetails);
                         }
+                        
+                        currentALMDefectFieldsValues.Add("screenshots", String.Join(",", defectSuggestion.ScreenshotFileNames););
```

Comment:
Codacy found an issue: [Remove this empty statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6043077)